litellm/model_prices_and_context_window_backup.json

```diff
@@ -1740,6 +1740,24 @@
         "supports_tool_choice": true,
         "supports_prompt_caching": true
     },
+    "groq/llama-3.3-70b-versatile": {
+        "max_tokens": 128000,
```

Comment:
@krrishdholakia Isn't it 8192?

https://github.com/BerriAI/litellm/issues/7034#issuecomment-2519403625

Reply:
fixed here @SmartManoj https://github.com/BerriAI/litellm/commit/a6a7accce60c243e5f9a814ef9e393ed8acdad66